common/src/main/java/com/wynntils/core/persisted/config/Config.java

```diff
@@ -49,7 +49,7 @@ public void setValue(T value) {
         this.userEdited = true;
     }
 
-    void restoreValue(Object value) {
+    public void setValueFromObject(Object value) {
```

Comment:
No-no. Use setValue. This is not to be changed. The reason why it is not used is commented above the setValue method. TLDR, the system was rewritten, but never fully migrated to.